### ISSUE TITLE
obsidian and electron_13

### ISSUE THREAD

**Issue by lucasew** [NixOS/nixpkgs#162402]
### Describe the bug
Obsidian (`pkgs.obsidian`) depends on `pkgs.electron_13` that is marked as insecure so it doesn't build without allowing insecure packages or doing some override shenanigans. Why the electron used by obsidian wasn't changed or bumped to a newer one?

### Steps To Reproduce
1. `nix-shell -p obsidian`
2. Package `electron-13.6.9 is marked as insecure refusing to evaluate
### Expected behavior
Download the required stuff and put in the PATH

### Notify maintainers
@ConradMearns  @zaninime 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
lucasew@acer-nix ~/.dotfiles 130$ nix-shell -p nix-info --run "nix-info -m"
warning: Git tree '/home/lucasew/.dotfiles' is dirty
warning: Git tree '/home/lucasew/.dotfiles' is dirty
 - system: `"x86_64-linux"`
 - host os: `Linux 5.16.7, NixOS, 22.05 (Quokka)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.6.0`
 - nixpkgs: `/nix/store/74hq7jg8hl7lmzh03wfrc1fijh1wrygk-nixpkgs`
```

**Comment by lucasew** [comment]
I just found a thing. When i override to electron_14 it raises this error
![image](https://user-images.githubusercontent.com/15693688/156229754-29fee2fe-2165-4522-8343-98f145b900cb.png)

It seems they removed this support on electron core and added as a plugin (`@electron/remote`)

**Comment by jyooru** [comment]
Electron has been updated in the insider build and this issue will be fixed when Obsidian makes a new public release:

https://forum.obsidian.md/t/update-electron-version-version-13-eol/33250

**Comment by psydvl** [comment]
I've created PR https://github.com/NixOS/nixpkgs/pull/160469 for 0.13.24, but since it's "placeholder build" we're waiting for next stable release

NB: and there is already opened issue for this situation... https://github.com/NixOS/nixpkgs/issues/158956

**Comment by jyooru** [comment]
Fixed in #160469
